### PR TITLE
Makes extradimensional oranges weirder.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Makes extradimensional oranges distill into mindbreaker toxin and changes their taste a bit.

## Why It's Good For The Game

o r a n g 

## Changelog
:cl:
add: orang
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -184,8 +184,8 @@
 	icon_state = "orang"
 	filling_color = "#FFA500"
 	juice_results = list(/datum/reagent/consumable/orangejuice = 0)
-	distill_reagent = /datum/reagent/consumable/ethanol/triple_sec
-	tastes = list("polygons" = 1, "oranges" = 1)
+	distill_reagent = /datum/reagent/reagent/toxin/mindbreaker
```
Comment (optimumtact) [inline code/modules/hydroponics/grown/citrus.dm:187]: reagent/reagent

Comment (TheSpaghetti) [reply code/modules/hydroponics/grown/citrus.dm]: Fixed.

**Comment by Fikou** [comment]
please make changelog better

**Comment by TheSpaghetti** [comment]
> please make changelog better

What do you mean by that?

**Comment by Fikou** [comment]
> > please make changelog better
> 
> What do you mean by that?

changelogs are what players see in game in the changelogs tab without any context, firstly "fixed "reagent/reagent"" is a fix to a bug that was introduced in this pr, which means that when the pr gets merged this wont matter at all to a player since its fixed already, so it shouldnt be there
secondly yes o r a n g is funny haha reddit gold

**Comment by TheSpaghetti** [comment]
> > > please make changelog better
> > 
> > 
> > What do you mean by that?
> 
> changelogs are what players see in game in the changelogs tab without any context, firstly "fixed "reagent/reagent"" is a fix to a bug that was introduced in this pr, which means that when the pr gets merged this wont matter at all to a player since its fixed already, so it shouldnt be there
> secondly yes o r a n g is funny haha reddit gold

orang

**Comment by TheSpaghetti** [comment]
the orang succeeded

**Pull request merged**